Cargo.toml

```diff
@@ -25,9 +25,10 @@ env_logger = { version = "0.10", default-features = false } # env_logger require
 futures-core = "0.3"
 futures-executor = "0.3"
 futures-util = { version = "0.3", default-features = false }
-hyper = { version = "0.14", default-features = false }
-http = { version = "0.2", default-features = false }
-isahc = { version = "1.4", default-features = false }
```

Comment:
We won't be able to continue supporting `isahc` with this upgrade?  Not a concern though, if this is not actively used in the Rust community :)